packages/easy-parallel/composer.json

```diff
@@ -8,7 +8,8 @@
         "react/child-process": "^0.6.5",
         "react/event-loop": "^1.3",
         "react/socket": "^1.12",
-        "symfony/console": "^6.2"
+        "symfony/console": "^6.2",
+        "fidry/cpu-core-counter": "^0.2.0"
```

Comment:
root composer.json may need update as well to fix phpstan class not found notice

Reply:
👍